.github/workflows/ci.yaml

```diff
@@ -54,10 +60,66 @@ jobs:
       TRIAL_BRANCH: ${{ needs.metadata.outputs.TRIAL_BRANCH }}
     secrets: inherit
 
+  publish-mealkit:
+    needs: [metadata, amd64, arm64]
+    if: "!canceled()" && github.event_name == 'schedule' && needs.metadata.outputs.PUBLISH == 'true'
+    strategy:
+      fail-fast: true
+      matrix:
+        - TAG: JAX
+          TARGET_IMAGE: jax
+        - TAG: PALLAS
+          TARGET_IMAGE: pallas
+        - TAG: PAX
+          TARGET_IMAGE: upstream_pax
+        - TAG: T5X
+          TARGET_IMAGE: upstream_t5x
+        - TAG: ROSSETA_PAX
+          TARGET_IMAGE: pax
+        - TAG: ROSSETA_T5X
+          TARGET_IMAGE: T5x
```

Comment:
I'm not sure casing matters, but should this be `t5x`

Reply:
Tags are indeed case-sensitive.

Reply:
This is the `image` part of the complete tag, which is required to be in lower case:
```
Error parsing reference: "Fedora:37" is not a valid repository/tag: invalid reference format: repository name must be lowercase
```